Gemfile

```diff
@@ -74,4 +74,5 @@ group :test do
   gem 'sqlite3', platforms: :mri
   gem 'activerecord-jdbcsqlite3-adapter', platforms: :jruby
   gem 'poltergeist'
+  gem 'tins', '1.6.0'
```

Comment:
That made 1.9 pass, but now there are different issues with JRuby. 

https://travis-ci.org/activeadmin/activeadmin/builds/149576226